app/README.md

```diff
@@ -33,8 +33,9 @@ can be found here: [config.example.js](public/config/config.example.js).
 | screenSharingFrameRate | The default screen sharing framerate. | `"nat"` | ``5`` |
 | simulcast | Enable or disable simulcast for webcam video. | `"boolean"` | ``true`` |
 | simulcastSharing | Enable or disable simulcast for screen sharing video. | `"boolean"` | ``false`` |
-| simulcastProfiles | Define different encodings for various resolutions of the video. | `"object"` | ``{  "320": [    {      "scaleResolutionDownBy": 1,      "maxBitRate": 150000    }  ],  "640": [    {      "scaleResolutionDownBy": 2,      "maxBitRate": 150000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 500000    }  ],  "1280": [    {      "scaleResolutionDownBy": 4,      "maxBitRate": 150000    },    {      "scaleResolutionDownBy": 2,      "maxBitRate": 500000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 1200000    }  ],  "1920": [    {      "scaleResolutionDownBy": 6,      "maxBitRate": 150000    },    {      "scaleResolutionDownBy": 3,      "maxBitRate": 500000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 3500000    }  ],  "3840": [    {      "scaleResolutionDownBy": 12,      "maxBitRate": 150000    },    {      "scaleResolutionDownBy": 6,      "maxBitRate": 500000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 10000000    }  ]}`` |
+| simulcastProfiles | Define different encodings for various resolutions of the video. | `"object"` | ``{  "320": [    {      "scaleResolutionDownBy": 1,      "maxBitRate": 250000    }  ],  "640": [    {      "scaleResolutionDownBy": 2,      "maxBitRate": 250000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 900000    }  ],  "1280": [    {      "scaleResolutionDownBy": 4,      "maxBitRate": 250000    },    {      "scaleResolutionDownBy": 2,      "maxBitRate": 900000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 3000000    }  ],  "1920": [    {      "scaleResolutionDownBy": 4,      "maxBitRate": 750000    },    {      "scaleResolutionDownBy": 2,      "maxBitRate": 1500000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 4000000    }  ],  "3840": [    {      "scaleResolutionDownBy": 4,      "maxBitRate": 1500000    },    {      "scaleResolutionDownBy": 2,      "maxBitRate": 4000000    },    {      "scaleResolutionDownBy": 1,      "maxBitRate": 10000000    }  ]}`` |
```

Comment:
Is there a reason you change simulcastProfiles?
I think this rolls back some testing/optimisations we made.